src/EventSubscriber/EventSubscriber.php

```diff
@@ -0,0 +1,133 @@
+<?php
+
+namespace Drupal\stanford_profile\EventSubscriber;
+
+use Drupal\Core\File\FileSystemInterface;
+use Drupal\Core\Logger\LoggerChannelFactoryInterface;
+use Drupal\Core\StreamWrapper\StreamWrapperManager;
+use Drupal\default_content\Event\DefaultContentEvents;
+use Drupal\default_content\Event\ImportEvent;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+
+/**
+ * Class EventSubscriber.
+ *
+ * @package Drupal\stanford_profile\EventSubscriber
+ */
+class EventSubscriber implements EventSubscriberInterface {
+
+  /**
+   * External site url to fetch the given file from.
+   */
+  const FETCH_DOMAIN = 'https://pookmish20191003.sites.stanford.edu';
```

Comment:
Shouldn't this be something that is going to stick around? This looks like something that could be deleted.

Reply:
ya. this was just temporary. what should the final domain be? i have to spin up a site before i change and merge this

Reply:
content.sites.stanford.edu was the winna.